content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_Host_Guardian_Service--Deployment_Guide/Getting_Help_and_Further_Reading.adoc

```diff
@@ -2,13 +2,13 @@
 
 Should you require assistance when using this guide to deploy YubiHSM 2 on Windows, start by referencing the product documentation and currently known issues:
 
-https://developers.yubico.com/[Yubico Developers website]
+link:../../../[Yubico Developers website]
 
-https://support.yubico.com/support/home[Yubico Support]
+link:https://support.yubico.com/support/home[Yubico Support]
 
-https://developers.yubico.com/YubiHSM2/[YubiHSM 2 Product Documentation]
+link:../../[YubiHSM 2 Product Documentation]
 
-https://developers.yubico.com/YubiHSM2/Releases/Known_issues.html[Known Issues and Limitations]
+link:../Releases/Known_issues.html[YubiHSM 2: Known Issues and Limitations]
```

Comment:
This needs a second ../ at the beginning to be correct

Reply:
https://github.com/Yubico/developers.yubico.com/pull/249/commits/e2d94b8a90103520efc39f2bdd924386e8190b80